Source/Core/Core/DSP/DSPCore.h

```diff
@@ -308,6 +308,9 @@ struct SDSP
 
   // This one doesn't really belong here.
   u8* cpu_ram;
+
+  // This is used by DSPJitRegCache to store EBP.
+  u64 ebp_store;
```

Comment:
The dispatcher (compiled in `DSPEmitter::CompileDispatcher`) saves all callee-save registers including rbp before jumping to compiled code.

The original intent of ebp_store seems to be to restore the original value of rbp before a function call. Is this correct? This appears to be unnecessary.